src/engraving/infrastructure/htmlparser.cpp

```diff
@@ -21,17 +21,127 @@
  */
 #include "htmlparser.h"
 
-#ifndef ENGRAVING_NO_QTEXTDOCUMENT
```

Comment:
One last thing: would be good to remove this define from `engraving/CMakeLists.txt`. After that, it's good!